src/config.ts

```diff
@@ -129,6 +131,7 @@ const DEFAULT_CONFIG: Partial<SnowpackConfig> = {
   },
   devOptions: {
     port: 8080,
+    open: false,
```

Comment:
lets default to true, this is what CRA does and most users prefer (although I definitely get why power users want to set to false).